<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHMModuleConfiguration</class>
 <widget class="QMainWindow" name="IHMModuleConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>application</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(0,0,0);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1111</width>
      <height>641</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="accueil">
     <widget class="QPushButton" name="lancer">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>480</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Lancer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="statistique">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>231</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Statistique</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageJollyJumpi">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>381</width>
        <height>281</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="affichagegif.qrc">:/images/logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>430</y>
        <width>231</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Z003</family>
        <pointsize>39</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#8f5902;&quot;&gt;Jolly JumPi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="configuration">
     <widget class="QSpinBox" name="nombreDeJoueur">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelNombreJoueur">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre de joueur</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelModeDeJeu">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Mode de jeu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="annuler">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>211</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmer">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>500</y>
        <width>211</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirmer</string>
      </property>
     </widget>
     <widget class="QLabel" name="joueur1">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Joueur 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="joueur2">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Joueur 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="joueur3">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>80</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Joueur 3</string>
      </property>
     </widget>
     <widget class="QLabel" name="joueur4">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>120</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Joueur 4</string>
      </property>
     </widget>
     <widget class="QLabel" name="joueur5">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>160</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Joueur 5</string>
      </property>
     </widget>
     <widget class="QLabel" name="joueur6">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>200</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Joueur 6</string>
      </property>
     </widget>
     <widget class="QLabel" name="joueur7">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>240</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Joueur 7</string>
      </property>
     </widget>
     <widget class="QLabel" name="joueur8">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>280</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Joueur 8</string>
      </property>
     </widget>
     <widget class="QComboBox" name="modeDeJeu">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>221</width>
        <height>131</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <italic>false</italic>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>Normal</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="attentePiste">
     <widget class="QLabel" name="connexionPiste1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>431</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Attente réponse de la piste  :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="annulerPiste">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>490</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelGif_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>180</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="connexionReussie">
     <widget class="QLabel" name="pistePrete1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>301</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>27</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Piste prête :</string>
      </property>
     </widget>
     <widget class="QLabel" name="connexionFait_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>190</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="affichagegif.qrc">:/images/icone-de-coche-verte.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="demarrer">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>490</y>
        <width>151</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Démarrer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="partieEnCours">
     <widget class="QPushButton" name="interrompre">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>510</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>Interrompre</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPartieEnCours">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>90</y>
        <width>351</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>La partie est en cours</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="finDePartie">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Partie Finis</string>
      </property>
     </widget>
     <widget class="QPushButton" name="menuAccueil">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>500</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="affichagegif.qrc"/>
 </resources>
 <connections/>
</ui>
